src/bidiMapper/utils/cdpClient.ts

```diff
@@ -17,18 +17,14 @@ import { log } from './log';
 const logCdp = log('cdp');
 
 import { EventEmitter } from './events';
+import { IServer } from './iServer';
```

Comment:
What do you think about changing the name of IServer to either IBinding or ITransport? I don't think "IServer" accurately describes what this class does. It represents that channel that client and server use to communicate. It's not the server itself.

Reply:
I find `server` is a bad naming as well, and `transport` suites better. But I don't want to do it in current PR, as the scope of this one grows too fast, and we end up with a huge un-reviewable PR.

Reply:
Improving the name as a follow-up patch sounds great!